common/djangoapps/edxmako/signals.py

```diff
@@ -0,0 +1,5 @@
+"""Signals related to the edxmako rendering methods."""
+from django.dispatch import Signal
+
+
```

Comment:
Can you add a comment here with your explanation of how one might use this signal (including any requirements on the response from the signal receiver)?